library/Notifications/Widget/ItemList/IncidentHistoryListItem.php

```diff
@@ -30,37 +27,38 @@ class IncidentHistoryListItem extends BaseListItem
 
     protected function assembleVisual(BaseHtmlElement $visual): void
     {
-        $incidentIcon = $this->getIncidentEventIcon();
-        if ($this->item->type === 'incident_severity_changed' || $this->item->type === 'opened') {
-            $content = new Icon($incidentIcon, ['class' => 'severity-' . $this->item->new_severity]);
-        } elseif ($this->item->type === 'rule_matched') {
-            $content = new IcingaIcon($incidentIcon, ['class' => 'type-' . $this->item->type]);
-        } else {
-            $content = new Icon($incidentIcon, ['class' => 'type-' . $this->item->type]);
+        $iconName = $this->getIconName();
```

Comment:
**Al**l icons returned by this, according to @flourish86's mockup, should be `fa-solid`.